src/omotes_simulator_core/entities/network_controller.py

```diff
@@ -44,6 +44,23 @@ def __init__(
         self.producers = producers
         self.consumers = consumers
         self.storages = storages
+        self._set_priority_from_marginal_costs()
+
```

Comment:
Fix the docstring of the above NetworkController.

Reply:
done